extension-requests/script.js

```diff
@@ -1320,6 +1455,35 @@ async function createExtensionCard(data, dev) {
   }
 }
 
+function shouldDisplayEditButton(assigneeId) {
+  return (
+    currentUserDetails &&
+    (assigneeId === currentUserDetails.id ||
+      currentUserDetails.roles.super_user)
+  );
+}
```

Comment:
Where are currentUserDetails coming from?
Can we make this a pure function?

Reply:
currentUserDetails is a global variable, we can make the function a pure function but that won't change anything, it's just we'll pass this global variable while making the call. So it will still depend on the global variable so is that any useful?

Reply:
So If you want to reuse this functionality in a different file, how will you do it?

Reply:
Okay then I'll pass currentUserDetails as  parameter, @prakashchoudhary07 does that work?

Reply:
Done